pkg/generators/models/testdata/cases/embedded_type/generated/model_top.go

```diff
@@ -5,12 +5,25 @@
 //     Version: 0.1.0
 package generatortest
 
+import (
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+)
+
 // Top is an object.
 type Top struct {
 	// Bar:
 	Bar Sub `json:"bar,omitempty"`
 }
 
+// Validate implements basic validation for this model
+func (m Top) Validate() error {
+	return validation.Errors{
+		"bar": validation.Validate(
+			m.Bar, validation.Required,
```

Comment:
why is it required if omitempty is true?

Reply:
because its not nullable, but I see the problem...

Reply:
how about `!nullable && isRequiredInSpec` as a criterium for adding validation.Required?

Reply:
ok. Now if something has omitempty on it, it is never validation.Required. Additionally it will be required if it is not nullable.